css/amp.css

```diff
@@ -78,6 +78,14 @@ body {
   right: 0;
 }
 
+.-amp-layout-flex-item {
+  position: relative;
+  flex: 1 1 0;
+  align-self: stretch;
+  height: 100%;
```

Comment:
Did you confirm if this always works as we expected? E.g. `flex-basis` takes over if parent is indeed `display: flex`?